bin/migration/tables/appaccess.py

```diff
@@ -0,0 +1,72 @@
+from .helpers import check_existing_record, parse_to_timestamp, audit_entry_creation, log_failed_imports
+from datetime import datetime
+import uuid
+
+class AppAccessManager:
+    def __init__(self, source_cursor):
+        self.source_cursor = source_cursor
+        self.failed_imports = set()
+
+    def get_data(self):
+        self.source_cursor.execute("SELECT * FROM public.users WHERE prerole != 'Level 3' OR prerole IS NULL")
+        return self.source_cursor.fetchall()
+
+    def migrate_data(self, destination_cursor, source_data):
+        batch_app_users_data = []
```

Comment:
@mazzopardi2 court_id is incorrect. need to be taken from groupoassignments table